samples/README.md

```diff
@@ -1,5 +1,46 @@
 # Samples
 
+## Build Instruction
+
+You can build the samples either separate or together.
+
+Firstly, build aws-iot-devices-sdk-cpp-v2 with following instructions:
+
+``` sh
+mkdir sdk-cpp-workspace
+cd sdk-cpp-workspace
+git clone --recursive https://github.com/aws/aws-iot-device-sdk-cpp-v2.git
+mkdir aws-iot-device-sdk-cpp-v2-build
+cd aws-iot-device-sdk-cpp-v2-build
+cmake -DCMAKE_INSTALL_PREFIX="<absolute path sdk-cpp-workspace dir>"  -DBUILD_DEPS=ON -DCMAKE_BUILD_TYPE=Release ../aws-iot-device-sdk-cpp-v2
```

Comment:
DCMAKE_INSTALL_PREFIX should be wherever they want it installed

Reply:
Should it be changed from the main README?

Reply:
yes